ui/app/templates/components/clients/dashboard.hbs

```diff
@@ -1,205 +1,25 @@
-<div class="box is-sideless is-fullwidth is-marginless is-bottomless">
-  <p class="has-bottom-margin-xl">
-    This data is presented by full month. If there is data missing, it’s possible that tracking was turned off at the time.
-    Vault will only show data for contiguous blocks of time during which tracking was on. Documentation is available
-    <DocLink @path="/docs/concepts/client-count">here</DocLink>.
-  </p>
-  <h1 data-test-client-count-title class="title is-6 has-bottom-margin-xs">
-    Billing start month
-  </h1>
-  <div class="is-flex-align-baseline">
-    <p class="is-size-6">{{this.startTimeDisplay}}</p>
-    <button type="button" class="button is-link" {{on "click" (fn (mut this.isEditStartMonthOpen) true)}}>
-      Edit
-    </button>
-  </div>
-  <p class="is-8 has-text-grey has-bottom-margin-xl">
-    This date comes from your license, and defines when client counting starts. Without this starting point, the data shown
-    is not reliable.
-  </p>
-  {{#if (eq @model.config.queriesAvailable false)}}
-    {{#if (eq @model.config.enabled "On")}}
-      <EmptyState
-        @title={{concat "No monthly history " (if this.noActivityDate "from ") this.noActivityDate}}
-        @message="There is no data in the monthly history yet. We collect it at the end of each month, so your data will be available on the first of next month."
-      />
-    {{else}}
-      <EmptyState
-        @title="Data tracking is disabled"
-        @message="Tracking is disabled, and no data is being collected. To turn it on, edit the configuration."
-      >
-        {{#if @model.config.configPath.canUpdate}}
-          <p>
-            <LinkTo @route="vault.cluster.clients.index" @query={{hash tab="config"}}>
-              Go to configuration
-            </LinkTo>
-          </p>
-        {{/if}}
-      </EmptyState>
-    {{/if}}
-  {{else}}
-    {{#if (eq @model.config.enabled "Off")}}
-      <AlertBanner data-test-tracking-disabled @type="warning" @title="Tracking is disabled">
-        Tracking is currently disabled and data is not being collected. Historical data can be searched, but you will need to
-        <LinkTo @route="vault.cluster.clients.edit">
-          edit the configuration
+<PageHeader as |p|>
```

Comment:
the former `dashboard.hbs` is now `history.hbs` (which is now `old-history.hbs`)

This new _and improved_ 🙃 `dashboard.hbs` contains the tabs and header for client counts. I had trouble using `{{outlet}}` (perhaps it had something to do with importing Route instead of ClusterRoute?!). Though it did work if the parent template was moved _outside_ of the `clients` directory. Opted for creating a separate component instead (per Jordan's recommendation, thanks again) mostly for simplicity and the config view wasn't exactly the same as history and current month